app/src/main/java/ar/com/wolox/android/example/ui/auth/login/AuthFragment.kt

```diff
@@ -8,16 +9,41 @@ class AuthFragment : WolmoFragment<FragmentLoginBinding, AuthPresenter>(), AuthV
     override fun layout() = R.layout.fragment_login
 
     override fun init() {
+        val sharedPref = activity?.getPreferences(Context.MODE_PRIVATE) ?: return
```

Comment:
You don't need to get the shared preferences. WoloxAndroidBootstrap provides a shared preferences class just for authentication purposes. It's called `UserSession`, feel free to add anything you need in that class!